### DISCUSSION TITLE
Question: Put button inside ListViewElement

### DISCUSSION THREAD

**Discussion by maltesermailo** [microsoft/microsoft-ui-xaml#7418] [Q&A]
Hi, I'm trying to use a ListView for display entries which each have 2 buttons.
Unfortunately, once you click the buttons they wont react. You have to select the listviewitem separately by clicking anywhere else on the element. I would like to use it so that the buttons instantly react. I have not yet found a solution, but tried searching for a solution for autoselecting the element on hover or a way to select it before the button gets called.
Any suggestions welcome!

**Comment by ghost1372** [comment]
here you can find some example
https://docs.microsoft.com/en-us/windows/apps/design/controls/commanding#command-experiences-using-the-standarduicommand-class

![image](https://user-images.githubusercontent.com/9213496/171354093-aeb10af1-7ba2-4ae7-a3db-5b5143930523.png)
